ApollosProject/apollos-prototype
# Fixes CardWrapper and Icon loading states

## DESCRIPTION
We have had some broken loading state functionality on a few of our ui primitives. There are probably others but this PR fixes `CardWrapper` (also imported as just `Card`) as well as `Icon`.

##### correct placeholders on Android
<img src="https://user-images.githubusercontent.com/2053547/61491810-01a3db80-a97e-11e9-9efb-524e33643a57.png" width="50%" />

### What does this PR do, or why is it needed?
In the past we have worked around this by explicitly passing `isLoading` when it should have been passed via context automatically. This shouldn't have any negative effects on components that had explicitly worked around this bug but it will make a few other places start working correctly. Namely the new `FeatureCard` and apparently the `UserAvatarHeader` 💥

### What design trade-offs/decisions were made?
N/A

### How do I test this PR?
Once #867 is merged the `isLoading` story on `FeaturedCard` is a great place to test this. Maybe click around a few other places in the app to confirm it's all working right.

### What automated tests did you write?
All these tests were already written. There are a bunch of other places where we are getting false positive tests because we aren't using out `renderWithApolloData` helper. Nothing is any more broken with that regard then it was before this PR so no worries there.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
